state/modelconfig.go

```diff
@@ -137,23 +137,32 @@ func (model *Model) modelConfigValues(modelCfg attrValues) (config.ConfigValues,
 }
 
 // UpdateModelConfigDefaultValues updates the inherited settings used when creating a new model.
-func (model *Model) UpdateModelConfigDefaultValues(attrs map[string]interface{}, removed []string, regionSpec *environs.RegionSpec) error {
+func (st *State) UpdateModelConfigDefaultValues(attrs map[string]interface{}, removed []string, regionSpec *environs.CloudRegionSpec) error {
 	var key string
 
 	if regionSpec != nil {
-		key = regionSettingsGlobalKey(regionSpec.Cloud, regionSpec.Region)
+		if regionSpec.Region != "" {
```

Comment:
Super minor nit - when you've got both an if and else clause I think it's more readable to order them so the condition is positive, so this would be 
```
if regionSpec.Region == "" {
    key = cloudGlobalKey(regionSpec.Cloud)
} else {
    key = regionSettingsGlobalKey(regionSpec.Cloud, regionSpec.Region)
}
```

(And the same for the outer if.)